code/controllers/subsystems/processing/electronics.dm

```diff
@@ -13,11 +13,14 @@
 	var/list/all_integrated_circuits = list()
 	var/list/printer_recipe_list = list()
 
+	var/cipherkey
```

Comment:
What´s the purpose of that?

Reply:
For ID scanners, etc., so that you can't just access the raw data and spoof it for all-access. It's encrypted with an XOR cipher.
If they can manage to guess the (rather long and complicated, as well as randomly-generated) access cipherkey, I think they deserve the relatively minor things they could do with it.

Reply:
As discussed on discord.
Due to the of xor it would be possible to calculate the encryption key with a bit of meta knowledge by just xoring the json object with the returned string from the reader component.
(You still need to properly pre/postprocess the string/object)

Reply:
This was removed, so it's fine now.